src/serializers/locations-serializer.js

```diff
@@ -7,18 +7,38 @@ import { paginate } from 'utils/paginator';
 import { apiBaseUrl, resourcePathLink, paramsLink } from 'utils/uri-builder';
 
 const locationResourceProp = openapi.components.schemas.LocationResource.properties;
+const serviceResourceProp = openapi.components.schemas.ServiceResource.properties;
 const locationResourceType = locationResourceProp.type.enum[0];
 const locationResourceKeys = _.keys(locationResourceProp.attributes.properties);
+const serviceResourceKeys = _.keys(serviceResourceProp.attributes.properties);
 const locationResourcePath = 'locations';
 
 /**
- * Serialize locationResources to JSON API
+ * Flattens the attributes of an included service for use in the serializer
  *
- * @param {object} rawLocation Raw data row from data source
+ * @param {object} services services attribute from the raw data row
+ * @returns {object} Object containing flattened service attributes
+ */
+const flattenAttributes = (services) => {
+  const newServices = [];
+  _.forEach(services, (service) => {
+    const newService = {
+      ...service,
+      ...service.attributes,
+    };
+    delete newService.attributes;
```

Comment:
Just my two cents, if you declare a new var then you should be able to only merge whatever you need. I don't recommend deleting attributes from it since it's very possible to cause potential bugs in the future.